spec/models/comment_spec.rb

```diff
@@ -67,6 +67,17 @@
 
   end
 
+  describe 'report_reasons' do
+
+    let(:comment) { FactoryGirl.build(:comment) }
+
+    it 'returns an array of strings' do
+      expect(comment.report_reasons).to be_an(Array)
+      expect(comment.report_reasons).to all(be_a(String))
```

Comment:
probably, was playing with this rather than have a "test" that a hard-coded array returns the same hard-coded array